src/components/Sidebar.tsx

```diff
@@ -43,58 +43,64 @@ export interface SidebarItem {
   name: string,
   icon: IconType,
   path: string,
+  regex: RegExp,
   role?: Role,
-
-  basePath?: string,
-  queryParam?: string,
 }
 
 const sidebarItems: SidebarItem[] = [
   {
     name: '我的会议',
     path: '/',
     icon: MdVideocam,
+    regex: new RegExp('^\/$|\/groups\/.'),
```

Comment:
use this format for compilation performance [1] and brevity:

```
regex: /^\/$|\/groups\/./
```

[1] https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_expressions